test/hotspot/jtreg/compiler/types/TestMeetXor.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8267332
+ * @summary Test meet on xor
+ * @library /test/lib /
+ * @run main/othervm compiler.types.TestMeetXor -XX::CompileCommand=dontinline,*::test*
+ */
+
+package compiler.types;
+
+import java.util.Random;
+import jdk.test.lib.Asserts;
+
+public class TestMeetXor {
+    public static void main(String[] args) throws Exception {
+        for (int i = 0; i < 10000; i++) {
+            testCase1E();
+            testCase2E();
+            testCase3E();
+
+            testCaseS();
+        }
+    }
+
+    static int[] count = new int[256];
+    static Random r = new Random(5);
```

Comment:
You should use `jdk.test.lib.Utils.getRandomInstance()` here which also prints the seed.